src/mca/rmaps/seq/rmaps_seq.c

```diff
@@ -368,6 +369,7 @@ static int prte_rmaps_seq_map(prte_job_t *jdata,
 
             /* move to next node */
             sq = (seq_node_t *) pmix_list_get_next(&sq->super);
+            PMIX_RELEASE(proc);
```

Comment:
I think the PMIX_RETAIN here makes it more clear that it is being added by reference to the prte_job_t and because the current reference to proc is no longer required, it is released. Makes it more readable perhaps.